components/cards/HSpaceCard.tsx

```
@@ -0,0 +1,235 @@
+    }
+
+    const {
+        banner,
+        avatar,
+        name,
+        tagline,
+        tags,
+        id,
+        category,
+    } = data;
+    const { profile } = useCeramicContext();
```

ESLint

[error] React Hook "useCeramicContext" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)